postprocessing/pp-g722.c

```diff
@@ -192,18 +170,17 @@ int janus_pp_g722_process(FILE *file, janus_pp_frame_packet *list, int *working)
 		JANUS_LOG(LOG_VERB, "Writing %d bytes out of %d (seq=%"SCNu16", step=%"SCNu16", ts=%"SCNu64", time=%"SCNu64"s)\n",
 			bytes, tmp->len, tmp->seq, diff, tmp->ts, (tmp->ts-list->ts)/8000);
 		/* Decode and save to wav */
-		AVPacket avpacket;
-		av_init_packet(&avpacket);
-		avpacket.data = (uint8_t *)buffer;
-		avpacket.size = bytes;
+		AVPacket *avpacket = av_packet_alloc();
```

Comment:
@lu-zero any reason why this allocation is inside the loop? I see for all other codecs you just do it once before the `while`, and just unref/initialize the packet at each loop, while for G.722 this is happening in the `while` instead. Is it because we're actually decoding something here, rather than just saving to a container?